<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>groupBox_7</class>
 <widget class="QGroupBox" name="groupBox_7">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>110</height>
   </rect>
  </property>
  <property name="title">
   <string>Well mode and binning</string>
  </property>
  <layout class="QFormLayout" name="formLayout_5">
   <property name="labelAlignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QRadioButton" name="rb_high">
       <property name="text">
        <string>High</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_low">
       <property name="text">
        <string>Low</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>Binning:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QRadioButton" name="rb_x11">
       <property name="text">
        <string>x11</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_x22">
       <property name="text">
        <string>x22</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_x44">
       <property name="text">
        <string>x44</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>Well mode:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
